manim/utils/units.py

```diff
@@ -0,0 +1,41 @@
+from functools import wraps
+
+from ..config import config
+
+PERCENT = "%"
+PIXELS = "px"
+MUNIT = "m"
```

Comment:
Anyone have a suggestion of a cleaner way to do this? Only use for these constants is for users when initializing a `Unit` object. I personally would prefer something that would look like `units.UNIT`.

Reply:
Alright, I'll take a look at it.

Reply:
Has become redundant because of the use of different classes for different units.